test/test_help.py

```diff
@@ -5,9 +5,13 @@
 import unittest
 
 import os
-from path import path
 from shovel import help
 from shovel.tasks import Shovel
+try:
```

Comment:
Generally, I would move this into a common helper package and then do the import from the helper package, but it's only being repeated twice for now.